index.html

```diff
@@ -0,0 +1,125 @@
+              </div>
+              <div class="form-group">
+                <label id="email-label" for="email">Email address</label>
+                <input type="email" class="form-control" placeholder="Enter Your Email" id="email" aria-describedby="emailHelp" required>
+              </div>
+              <div class="form-group">
+                <label id="number-label" for="email">Age</label>
+                <input type="number" class="form-control" placeholder="Enter Your Age" min="10" max="70" id="number" aria-describedby="emailHelp" required>
+              </div>
+              <div class="form-group">
+                <label for="dropdown">Marital Status</label>
+                <select class="form-control" id="dropdown" required>
+                  <option>Single</option>
+                  <option>Married</option>
+                  <option>Divorced</option>
+                </select>
+              </div>
+              <label id="number-label" for="email">Gender</label>
+              <div class="form-check">
+                <input class="form-check-input" type="radio" name="gender" id="male" value="male" checked>
+                <label class="form-check-label" for="male">
+                  Gender: Male
+                </label>
+              </div>
+              <div class="form-check">
+                <input class="form-check-input" type="radio" name="gender" id="female" value="female">
+                <label class="form-check-label" for="female">
+                  Gender: Female
+                </label>
+              </div>
+
+              <label id="number-label" for="email">Preferred Programming Language</label>
+
+              <div class="form-check">
+                <input class="form-check-input" type="checkbox" value="javascript" id="defaultCheck1">
+                <label class="form-check-label" for="defaultCheck1">
+                  Javascript
+                </label>
+              </div>
+              <div class="form-check">
+                <input class="form-check-input" type="checkbox" value="php" id="defaultCheck1">
+                <label class="form-check-label" for="defaultCheck1">
+                  Php
+                </label>
+              </div>
+              <div class="form-check">
+                <input class="form-check-input" type="checkbox" value="Python" id="defaultCheck1">
+                <label class="form-check-label" for="defaultCheck1">
+                  Python
+                </label>
+              </div>
+              <div class="form-group">
+                <label for="exampleFormControlTextarea1">Description</label>
+                <textarea class="form-control" id="text-area" rows="3"></textarea>
+              </div>
+              <div class="form-group">
+                 <button id="submit" class="btn btn-primary">submit</button>
+              </div>
+
+
+
```

Comment:
Please do not leave unnecessary spaces in your code . Leaving this spaces makes your code bulky and not presentable